### DISCUSSION TITLE
Examples of Unicorn "in the wild"

### DISCUSSION THREAD

**Discussion by adamghill** [django-commons/django-unicorn#101] [Show and tell]
I'd love to start compiling a list of places `Unicorn` has been used (even though I realize there probably aren't many, yet!).

- https://www.django-unicorn.com uses Unicorn to show example components

**Comment by adamghill** [comment]
[python-utils](https://www.python-utils.com) uses `Unicorn` for all of its examples and the source is available at https://github.com/adamghill/python-utils.com.

**Comment by liamlawless35** [comment]
[www.unendingstory.com](www.unendingstory.com) is completely built off of django-unicorn. How do I know? am dev of said website :/

**Reply by LexxLuey** [reply]
the website is expired

**Comment by adamghill** [comment]
https://devmarks.io/, _Bookmarking for Developers_, is built using `Unicorn`, https://pypi.org/project/django-fbv/, and https://pypi.org/project/coltrane-web/.

**Comment by adamghill** [comment]
I know [@zurtri ](https://github.com/zurtri) uses `Unicorn` for https://horserecords.info/foaling_date_calculator/! 🐴

**Reply by nerdoc** [reply]
Does not work any more...

**Comment by adamghill** [comment]
I'm using `Unicorn` for https://fedigest.adamghill.com/ which aggregates the recent popular posts from your Mastodon timeline.

**Reply by nerdoc** [reply]
Server is not available any more.

**Reply by adamghill** [reply]
I moved it to https://fediview.com.
